packages/canvas-node/contract.js

```diff
@@ -0,0 +1,128 @@
+export const chains = [
+  'eip155:*',
+  'cosmos:*',
+  'solana:*',
+  'polkadot:*',
+  'near:*',
+];
+
+export const models = {
+  threads: {
+    id: 'string',
+    community: 'string',
+    creator: 'string',
+    title: 'string',
+    body: 'string',
+    link: 'string',
+    updated_at: 'datetime',
+    indexes: ['creator'],
+  },
+  comments: {
+    id: 'string',
+    creator: 'string',
+    thread_id: 'string',
+    body: 'string',
+    parent_comment_id: 'string',
+    updated_at: 'datetime',
+    indexes: ['thread_id', 'creator'],
+  },
+  thread_reactions: {
+    id: 'string',
+    thread_id: 'string',
+    creator: 'string',
+    value: 'integer',
+    updated_at: 'datetime',
+    indexes: ['thread_id', 'creator'],
+  },
+  comment_reactions: {
+    id: 'string',
+    comment_id: 'string',
+    creator: 'string',
+    value: 'integer',
+    updated_at: 'datetime',
+    indexes: ['comment_id', 'creator'],
+  },
+};
+
+export const actions = {
```

Comment:
Are these the actions that verify.js verifies against?

Reply:
Docs here: https://docs.canvas.xyz/docs/canvas/packages/interfaces#actions

Reply:
Yes, although I'd like to update the Canvas packages again to support the updated Magic/Launchpad signing formats, rather than overloading methods as we do now.